usecases/cluster/delegate_test.go

```diff
@@ -202,7 +202,7 @@ func TestDelegateSort(t *testing.T) {
 	delegate.set("N2", NodeInfo{DiskUsage{Available: GB + 128}, now})
 	delegate.set("N3", NodeInfo{DiskUsage{Available: GB + 512}, now})
 	// one block more
-	delegate.set("N4", NodeInfo{DiskUsage{Available: GB + 4096}, now})
+	delegate.set("N4", NodeInfo{DiskUsage{Available: GB + 1<<25}, now})
```

Comment:
can we use direct number instead of bit shifting to be easier to reason about by just looking at the number  ?